chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/SearchRequest.java

```diff
@@ -144,4 +147,62 @@ public Boolean getHasHadSurveillance() {
 	public void setHasHadSurveillance(Boolean hasHadSurveillance) {
 		this.hasHadSurveillance = hasHadSurveillance;
 	}
+	
+	@Override
+	public int hashCode() {

```

Comment:
Works for me. Do you have a link to the algorithm? I'm curious to see the reasoning behind it, though I don't have an issue with it here